packages/schematics/angular/migrations/update-11/replace-ng-packagr-builder.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Rule, chain } from '@angular-devkit/schematics';
+import { NodeDependencyType, addPackageJsonDependency, removePackageJsonDependency } from '../../utility/dependencies';
+import { latestVersions } from '../../utility/latest-versions';
+import { updateWorkspace } from '../../utility/workspace';
+import { Builders, ProjectType } from '../../utility/workspace-models';
+
+export default function (): Rule {
+  return chain([
+    updateWorkspace(workspace => {
+      for (const [, project] of workspace.projects) {
+        if (project.extensions.projectType !== ProjectType.Library) {
+          continue;
+        }
+
+        for (const [, target] of project.targets) {
+          if (target.builder === Builders.DeprecatedNgPackagr) {
+            target.builder = Builders.NgPackagr;
+          }
+        }
+      }
+    }),
+    host => {
+      removePackageJsonDependency(host, '@angular-devkit/build-ng-packagr');
```

Comment:
Not for this PR but we should consider turning these dependency helpers into actual rules so the anonymous function isn't needed.

Reply:
yeah I think that's a good idea.